core/src/bundle/usBundleResource.cpp

```diff
@@ -281,20 +281,14 @@ void* BundleResource::GetData() const
 {
   if (!IsValid()) return nullptr;
 
-  void* data = d->archive->GetResourceContainer()->GetData(d->stat.index);
-  if (data == nullptr)
-  {
-    US_WARN << "Error uncompressing resource data for " << this->GetResourcePath() << " from "
-            << d->archive->GetBundleLocation();
-  }
-  return data;
+  return d->archive->GetResourceContainer()->GetData(d->stat.index);
```

Comment:
If plumbing is the only concern, I think we could easily do
```
GetPrivate(GetBundleContext())->GetLogSink()
```
without any plumbing, right?